src/main/java/lotto/viewer/OutputView.java

```diff
@@ -41,6 +50,23 @@ private void printSingleResult(LottoRank key, int value) {
             return;
         }
         System.out.printf(REGULAR_RESULT_EXPRESSION + System.lineSeparator(),
-            (int) key.getMatchingCount() ,key.getPrizeMoney(),value);
+            (int) key.getMatchingCount(), key.getPrizeMoney(), value);
+    }
+
+    public void printMoneyException(MoneyException moneyException) {
+        System.out.println(moneyException.getMessage());
+    }
+
+    public void printLottoAnnouncementException(
+        LottoAnnouncementException lottoAnnouncementException) {
+        System.out.println(lottoAnnouncementException.getMessage());
+    }
+
+    public void printPieceException(PieceException pieceException) {
+        System.out.println(pieceException.getMessage());
+    }
+
+    public void printLottoException(LottoException lottoException) {
+        System.out.println(lottoException.getMessage());
     }
```

Comment:
```suggestion
    public void printMoneyException(Exception e) {
        System.out.println(e.getMessage());
    }
```

해당 메서드들을 나누신 이유가 있을까요!?